### DISCUSSION TITLE
How to write back config load via `viper.Unmarshal(&config)`?

### DISCUSSION THREAD

**Discussion by hyorigo** [spf13/viper#1078] [Q&A] [answered]
Usually, we change the value with `viper.Set()` and then viper.WriteConfig() to write back.

What about the config load via `viper.Unmarshal(&config)`? How to set the config and write back?

**Comment by sagikazarmark** [comment, marked as answer]
It's not currently possible at the moment and I doubt this will ever be a feature. `WriteConfig` is not supposed to be used as a means to save the configuration changes, rather to dump a state so you can create eg. a default config file.
